src/components/SelectBox.res

```diff
@@ -1169,8 +1170,26 @@ module RenderListItemInBaseRadio = {
       })
       ->React.array
 
+    let sidebarScrollbarCss = `
+      @supports (-webkit-appearance: none) {
+        .sidebar-scrollbar {
+          scrollbar-width: thin !important;
+          scrollbar-color: #8a8c8f;
+        }
+
+        .sidebar-scrollbar::-webkit-scrollbar-thumb {
+          background-color: #8a8c8f;
+          border-radius: 3px;
+        }
+
+        .sidebar-scrollbar::-webkit-scrollbar-track {
+          display: none;
+        }
+      }
+    `
     <>
-      <div className=""> {dropdownList} </div>
+      <div className={`${customScrollStyle}  sidebar-scrollbar`} >
+       <style> {React.string(sidebarScrollbarCss)} </style> {dropdownList} </div>
```

Comment:
Can you add a condition to show scrollbar (the <style> tag)  only if `customScrollStyle` is not empty string?
Same for sidebar-scrollbar class, add that class only if `customScrollStyle` is present